build/pkgs/mathjax/distros/gentoo.txt

```diff
@@ -1 +1 @@
-">=dev-libs/mathjax-3"
+>=dev-libs/mathjax-3
```

Comment:
`>=` ?

Reply:
What about it?

Reply:
Oops, sorry. It's OK. I never saw such syntax, but it's valid